web_listview_custom_column/models/ir_ui_view.py

```diff
@@ -39,6 +39,21 @@ def custom_column(self, diff):
                     element.getnext().addnext(element)
                     break
         elif diff['operation'] == 'reset':
+            # if empty, then search if a global view has been set on this
+            # and prompt user who tries to delete to talk to user who
+            # created those custom views.
+            if not customized_view:
+                views = self.search([
+                    ('model', '=', self.model),
+                    ('mode', '=', 'extension'),
+                    ('user_ids', '=', False),
+                    ('create_uid', '!=', self.env.user.id,),
+                    ])
+                raise exceptions.ValidationError(_(
+                    'Users %s have created views that apply to everyone. \n '
+                    'If you want to change this view you need to notify \n'
+                    'them to reset the views they have set by clicking this \n'
+                    'button') % (views.mapped('create_uid.name', )))
```

Comment:
you don't need the brackets around `views....`, and also no comma there

Reply:
wait, this will show `[user1, user2]` when there are multiple users (which probably never happens, but anyways). Better wrap in `', '.join()`